docs/system/k8s-metrics.md

```diff
@@ -60,6 +60,31 @@ This metric is [recommended][MetricRecommended].
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `k8s.pod.memory.usage`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.k8s.pod.memory.usage -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `k8s.pod.memory.usage` | Gauge | `By` | Memory usage of the Pod [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
That's mostly intentional so far to avoid any conflict with the resource/entities work. We definitely need to take care of it though. I have filed https://github.com/open-telemetry/semantic-conventions/issues/1421 to keep track of it.